packages/components/select-v2/src/select-dropdown.tsx

```diff
@@ -190,7 +220,7 @@ export default defineComponent({
     }
 
     const onEscOrTab = () => {
-      select.expanded = false
+      select.expanded.value = false
```

Comment:
Is there a bug here?

Reply:
Yep, it should indeed not be modified in this PR, causing confusion. I will revert it.